### PR TITLE
bower packages

### PR DESCRIPTION
**Pull request by anthonyettinger**
Move all vendor libraries under bower control

- update libraries to known working versions
- vendor libs are now ignored by git

### CODE REVIEW

**Comment by anthonyettinger** [comment]
@seanbrookes @jtary PTAL

**Comment by jtary** [comment]
LGTM

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "strong-arc",
+  "version": "1.3.0",
```
Comment (seanbrookes) [inline bower.json]: how is this value maintained?  We are technically on Arc v 1.4

**Comment by seanbrookes** [comment]
one comment/question - other than that"

LGTM

**Comment by anthonyettinger** [comment]
test please

**Comment by rmg** [comment]
CI is failing because strong-mesh-client is failing to install:
```
> strong-mesh-client@1.6.1 install /mnt/jenkins/workspace/strong-arc.pr/eb78835e/node_modules/strong-mesh-client
> node proxy/build-client.js

uglify-js failed on node_modules/strong-mesh-models/node_modules/http-auth/node_modules/htpasswd/lib/htpasswd.js : Unexpected character '#' (line: 1, col: 0, pos: 0)

Error
    at new JS_Parse_Error (/mnt/jenkins/workspace/strong-arc.pr/eb78835e/node_modules/strong-mesh-client/node_modules/minifyify/node_modules/uglify-js/lib/parse.js:196:18)
    at js_error (/mnt/jenkins/workspace/strong-arc.pr/eb78835e/node_modules/strong-mesh-client/node_modules/minifyify/node_modules/uglify-js/lib/parse.js:204:11)
    at parse_error (/mnt/jenkins/workspace/strong-arc.pr/eb78835e/node_modules/strong-mesh-client/node_modules/minifyify/node_modules/uglify-js/lib/parse.js:314:9)
    at Object.next_token [as input] (/mnt/jenkins/workspace/strong-arc.pr/eb78835e/node_modules/strong-mesh-client/node_modules/minifyify/node_modules/uglify-js/lib/parse.js:567:9)
    at next (/mnt/jenkins/workspace/strong-arc.pr/eb78835e/node_modules/strong-mesh-client/node_modules/minifyify/node_modules/uglify-js/lib/parse.js:666:25)
    at Context.parse (/mnt/jenkins/workspace/strong-arc.pr/eb78835e/node_modules/strong-mesh-client/node_modules/minifyify/node_modules/uglify-js/lib/parse.js:652:15)
    at /mnt/jenkins/workspace/strong-arc.pr/eb78835e/node_modules/strong-mesh-client/node_modules/minifyify/node_modules/uglify-js/tools/node.js:81:33
    at Array.forEach (native)
    at Object.exports.minify (/mnt/jenkins/workspace/strong-arc.pr/eb78835e/node_modules/strong-mesh-client/node_modules/minifyify/node_modules/uglify-js/tools/node.js:76:15)
    at finish (/mnt/jenkins/workspace/strong-arc.pr/eb78835e/node_modules/strong-mesh-client/node_modules/minifyify/lib/minifier.js:175:28)
unable to generate browser bundle: Cannot find module './program' from '/mnt/jenkins/workspace/strong-arc.pr/eb78835e/node_modules/strong-mesh-client/node_modules/strong-mesh-models/node_modules/http-auth/node_modules/htpasswd/lib'
```

**Comment by rmg** [comment]
@kraman is working on a fix in strong-mesh-models

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by anthonyettinger** [comment]
test please

**Comment by anthonyettinger** [comment]
test please

**Comment by rmg** [comment]
@slnode test please

**Pull request merged**